### PR TITLE
Use ansi-to-html to render ANSI characters in logs

### PR DESCRIPTION
**Pull request by vincent-psarga**
## Summary

This is linked to issue #1057 - it uses `ansi-to-html` to turn ANSI characters into colored text.

A few notes:
 - it only works with strings that have been logged using the `log` method. The idea its to avoid getting colorised output due to some file content for example, but that can be discussed/changed.
 - I did not manage to get the TypeScript declaration from `ansi-to-html`, so there's a copy of their declaration file in `types.d.ts`- there is certainly a better way to do this but I did not find it :/

## How Has This Been Tested?

One unit test with Mocha and also an example in storybook.

## Screenshots (if appropriate):

Screenshot 2020-06-15 at 14 36 12

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).

## Checklist:

- [ ] The change has been ported to Java.
- [ ] The change has been ported to Ruby.
- [x] The change has been ported to JavaScript.
- [ ] The change has been ported to Go.
- [ ] The change has been ported to .NET.
- [x] I've added tests for my code.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the CHANGELOG accordingly.

### CODE REVIEW

**Comment by aslakhellesoy** [comment]
Sorry for duplicating this in #1062 

I suggest keeping from mine:
- TCK updates

Remove from yours:
- The `.d.ts` - use `@tsignore`

**Comment by vincent-psarga** [comment]
What do you mean by the TCK updates exactly ?
Also I think the icons added previsously were nice to show that it's an attachment.

File: react/javascript/src/components/gherkin/Attachment.tsx [hunk 1]
```diff
@@ -13,10 +15,12 @@ const Attachment: React.FunctionComponent<IProps> = ({ attachment }) => {
...
   } else if (attachment.mediaType.match(/^video\//)) {
     return video(attachment)
+  } else if (attachment.mediaType == 'text/x.cucumber.log+plain') {
+    return text(attachment, prettyANSI, true)
   } else if (attachment.mediaType.match(/^text\//)) {
-    return text(attachment, (s) => s)
+    return text(attachment, (s) => s, false)
   } else if (attachment.mediaType.match(/^application\/json/)) {
-    return text(attachment, prettyJSON())
+    return text(attachment, prettyJSON(), false)
```
Comment (aslakhellesoy) [inline react/javascript/src/components/gherkin/Attachment.tsx:23]: Can we change this from `prettyJSON()` to `prettyJSON` like I had done in my PR?

Comment (vincent-psarga) [reply react/javascript/src/components/gherkin/Attachment.tsx:23]: yes sure :)

**Comment by aslakhellesoy** [comment]
> What do you mean by the TCK updates exactly ?
> Also I think the icons added previsously were nice to show that it's an attachment.

I meant this @vincent-psarga: https://github.com/cucumber/cucumber/pull/1062/files#diff-7e99e425a6fd63d3ca507b086ed233d3

**Comment by aslakhellesoy** [comment]
Yes the attachment icons are nice - I deleted them by accident

**Pull request merged**